src/components/Modal/ModalDelete/ModalDelete.jsx

```diff
@@ -0,0 +1,30 @@
+import 'src/components/Modal/ModalDelete/styles.scss';
```

Comment:
if the styles are in the same directory as the component, then you can omit the absolute path. Just use a dot (.)